### DISCUSSION TITLE
removing Nuxt Prometheus Module problem

### DISCUSSION THREAD

**Discussion by andamansunda** [nuxt/nuxt#27369] [Questions]
I recently installed the Nuxt Prometheus module to try to sort out a memory leak problem. For technical reasons involving our deployment, I realized that we did not need the module. So I removed the entry from the modules section of the nuxt.config file, and I then uninstalled the package. The problem is that it was not entirely removed. When I have the development environment up, I then get a batch of messages " WARN  [Vue Router warn]: No match found for location with path "/metrics"" in the terminal and these messages come up again every 15 seconds or so. This makes development rather annoying. 
So my questions are as follows: 

- Is there a way to remove these messages?
- Is there a requirement for the producers and publishers of modules promoted on the Nuxt website to be reachable for troubleshooting issues? I found one email address to contact the publisher of this module, but I have not received a response to my email. 

Thanks, Andaman

**Comment by mfazail** [comment]
I think there might be compiled js of the that module has left in .nuxt folder, please delete .nuxt folder and start the development server to generate it again

**Reply by andamansunda** [reply]
Thank you!

**Reply by andamansunda** [reply]
That would have been great, but it didn't solve the problem. I deleted the .nuxt folder, tried npm run dev again, and I still get the messages. I then went ahead and deleted the pnpm-lock file, deleted the node_modules folder, ran pnpm install and then npm run dev, and the Vue Router warn message still comes up every 15 seconds.
